docker/test/integration/features/mqtt.feature

```diff
@@ -0,0 +1,66 @@
+Feature: Sending data to using MQTT streaming platform using PublishMQTT
+  In order to send and receive data via MQTT
+  As a user of MiNiFi
+  I need to have PublishMQTT and ConsumeMQTT processors
+
+  Background:
+    Given the content of "/tmp/output" is monitored
+
+  Scenario: A MiNiFi instance transfers data to an MQTT broker
+    Given a GetFile processor with the "Input Directory" property set to "/tmp/input"
+    And a file with the content "test" is present in "/tmp/input"
+    And a PublishMQTT processor set up to communicate with an MQTT broker instance
+    And a PutFile processor with the "Directory" property set to "/tmp/output"
+    And the "success" relationship of the GetFile processor is connected to the PublishMQTT
+    And the "success" relationship of the PublishMQTT processor is connected to the PutFile
+
+    And an MQTT broker is set up in correspondence with the PublishMQTT
+
+    When both instances start up
+    Then a flowfile with the content "test" is placed in the monitored directory in less than 60 seconds
+    And the MQTT broker has 1 log line matching "Received PUBLISH from .*testtopic.*\\(4 bytes\\)"
+
+  Scenario: A MiNiFi instance tries to transfer data to a non-existent MQTT broker
```

Comment:
Is the test file not routed to failure because PublishMQTT will throw in `onSchedule()`?  If so, then the name of this scenario could be better, eg. "If the MQTT broker is not available, the PublishMQTT processor will not start".

Reply:
No, it is because PublishMQTT will do nothing and wait for the broker to appear. Then it will forward it. I'm open to discussion if this is a desired behavior.

Reply:
OK, I get it -- if the processor can't connect to the broker, it won't even pick up any flow files from the queue.  I would change the title of this scenario to better explain this, e.g. "If the MQTT broker does not exist, then no flow files are processed".

This approach could be improved (e.g. with a configurable timeout), but definitely not in this pull request.